Dragonflight/WarriorArms.lua

```diff
@@ -1,17 +1,17 @@
 -- WarriorArms.lua
--- September 2022
+-- October 2022
+-- Updated for PTR Build 46181
+-- Last Modified 10/20/2022 18:15 UTC
 
 if UnitClassBase( "player" ) ~= "WARRIOR" then return end
 
 local addon, ns = ...
 local Hekili = _G[ addon ]
 local class, state = Hekili.Class, Hekili.State
 
-local FindPlayerAuraByID = ns.FindPlayerAuraByID
-
 local spec = Hekili:NewSpecialization( 71 )
 
-local base_rage_gen, arms_rage_mult = 1.75, 4.000
+local base_rage_gen, arms_rage_mult, norm_weapon_speed = 1.75, 4.000, 3.6
```

Comment:
I'm not sure this accomplishes anything useful.  If you're wielding a 3.6 and have haste that brings your swing timer down to 2.5 (i.e., 44% haste), then `state.haste` will be `1 / 1.44` which equals `0.694`.  So the value calculation down below will be ... `* base_rage_gen * arms_rage_mult * 2.5 / 0.694`.  `2.5 / 0.694` will get you back to 3.6 for the value calculation.  

Were you seeing different rage generation values?  Remember, these are virtual rage gains that the addon is forecasting, not real rage gains to which the addon simply reacts.

Reply:
I originally misread the divisor at the end as a multiplication, and in my calculations was coming up wrong, therefore it caused me to go down that rabbit hole. 

Reverted in https://github.com/Hekili/hekili/pull/1670/commits/c8e046d4407cec40f706237161b27d87f166ecd5 to previous formula.